### DISCUSSION TITLE
Integration with GIS QGIS Desktop

### DISCUSSION THREAD

**Discussion by jonnyforestGIS** [duckdb/duckdb-spatial#107] [Ideas]
Hi to all,
QGIS Desktop ([https://www.qgis.org/en/site/](https://www.qgis.org/en/site/)) is one of the open source GIS Desktop used by users.

I wonder if duckdb and duckdb_spatial could be excellent addition to both projects. QGIS uses as well GEOS, GDAL and PROJ.

See the repo here: [https://github.com/qgis/QGIS](https://github.com/qgis/QGIS)

Cheers and congrats to Duckdb project and this new extension.

**Comment by Maxxen** [comment]
Hi!
Creating a QGIS toolbox with DuckDB is an excellent idea, but probably not something I'm going to look into for a while until we have some more functionality/optimizations present. But if anybody else wants to give it a shot you have my blessing. While creating a full-blown toolbox might be a lot of work, we should be able to leverage DuckDBs great python bindings for creating a PyQGIS plugin pretty easily.

**Reply by jonnyforestGIS** [reply]
@Maxxen the company Oslandia that contributes to QGIS project make a plugin with this integration. Check in here:

https://gitlab.com/Oslandia/qgis/qduckdb

Cheers.

**Comment by florentfougeres** [comment]
Hi to all, 
I work at [Oslandia](https://oslandia.com/) and we've just developed a [QGIS plugin](https://plugins.qgis.org/plugins/qduckdb/) that lets you read duckdb spatial data in QGIS. You can find more information in this [article](https://oslandia.com/en/2023/11/16/un-plugin-duckdb-dans-qgis/) or in the [gitlab](https://gitlab.com/Oslandia/qgis/qduckdb) project.

**Reply by jonnyforestGIS** [reply]
Thank you @florentfgrs for your work. Cheers. I think we can close this issue.
